rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -1154,6 +1154,8 @@ extern "C" rmw_ret_t rmw_init(const rmw_init_options_t * options, rmw_context_t
 
   context->instance_id = options->instance_id;
   context->implementation_identifier = eclipse_cyclonedds_identifier;
+  context->actual_domain_id =
+    RMW_DEFAULT_DOMAIN_ID != options->domain_id ? options->domain_id : 0u;
```

Comment:
@ivanpauno [this line](https://github.com/ros2/rmw_cyclonedds/pull/208/files#diff-d03c99cafce317e1a93de46e787e0235R944) has to change too.

Reply:
@hidmic can you clarify which line you're referring to?
The hyperlink redirects to this same comment.

Reply:
Ok, I think I guessed which line was: https://github.com/ros2/rmw_cyclonedds/pull/208/commits/a8b45bfd2ccf0639bf769506422e3e38b63dcec3

Reply:
Uh, that's so weird. It's almost like if Github's line hyperlinks within a PR diff are broken. Anyhow, you guessed right :)

Reply:
> It's almost like if Github's line hyperlinks within a PR diff are broken

Yeah, links within a PR diff are broken in my experience after the review is sent.
Opening the file and getting the permalink always works correctly.